src/app/core/components/generic-mod-list/generic-mod-list.component.html

```diff
@@ -23,9 +23,10 @@
       </div>
       <div class="mod-list-container">
         <strong class="mod-name one-line-clamp">{{ mod.name }}</strong>
+        <div class="mod-version">Last update: {{ getLastUpdateText(mod.lastUpdate) }}</div>
```

Comment:
We should not use functions in the template. I have done the same on line 30 but we should avoid that. Angular Change Detection cant "cache" results from functions and will execute them every change detechtion what would be a lot

Reply:
I see, I am now saving it as the built string to display.